<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QMainWindow" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="ObjTreeWgt" name="objsList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>390</width>
      <height>411</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <column>
     <property name="text">
      <string>World</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>979</x>
      <y>0</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="FileListWgt" name="fileList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>390</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>392</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionRun"/>
    <addaction name="actionImport_Resource"/>
    <addaction name="separator"/>
    <addaction name="actionBuild"/>
    <addaction name="actionClose_project"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>&amp;Create</string>
    </property>
    <addaction name="actionCreateScene"/>
    <addaction name="actionCreateMaterial"/>
    <addaction name="actionCreateScript"/>
    <addaction name="actionLocStringFile"/>
   </widget>
   <widget class="QMenu" name="menuObjects">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionNew_Object"/>
    <addaction name="actionNew_Cube"/>
    <addaction name="actionNew_Light"/>
    <addaction name="actionNew_Tile"/>
    <addaction name="actionNew_Terrain"/>
    <addaction name="actionNew_Audio_source"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Cameras"/>
    <addaction name="actionRender_settings"/>
    <addaction name="actionPhysics_Settings"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionInspectorWin"/>
    <addaction name="actionConsole_log"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCreate"/>
   <addaction name="menuObjects"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionCreateScene">
   <property name="text">
    <string>&amp;Scene</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Sa&amp;ve As</string>
   </property>
  </action>
  <action name="actionNew_Object">
   <property name="text">
    <string>&amp;New Object</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="text">
    <string>Close project</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionToggle_Cameras">
   <property name="text">
    <string>Toggle Cameras</string>
   </property>
  </action>
  <action name="actionNew_Cube">
   <property name="text">
    <string>New Cube</string>
   </property>
  </action>
  <action name="actionNew_Light">
   <property name="text">
    <string>New Light</string>
   </property>
  </action>
  <action name="actionNew_Tile">
   <property name="text">
    <string>New Tile</string>
   </property>
  </action>
  <action name="actionCreateMaterial">
   <property name="text">
    <string>Material</string>
   </property>
  </action>
  <action name="actionCreateScript">
   <property name="text">
    <string>Script</string>
   </property>
  </action>
  <action name="actionRender_settings">
   <property name="text">
    <string>Render settings</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionNew_Terrain">
   <property name="text">
    <string>New Terrain</string>
   </property>
  </action>
  <action name="actionPhysics_Settings">
   <property name="text">
    <string>Physics Settings</string>
   </property>
  </action>
  <action name="actionImport_Resource">
   <property name="text">
    <string>Import Resource</string>
   </property>
  </action>
  <action name="actionLocStringFile">
   <property name="text">
    <string>Localized String File</string>
   </property>
  </action>
  <action name="actionInspectorWin">
   <property name="text">
    <string>Inspector</string>
   </property>
  </action>
  <action name="actionConsole_log">
   <property name="text">
    <string>Console log</string>
   </property>
  </action>
  <action name="actionNew_Audio_source">
   <property name="text">
    <string>New Audio source</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjTreeWgt</class>
   <extends>QTreeWidget</extends>
   <header>../../../../../../code/ProjEd/headers/ProjectEdit.h</header>
  </customwidget>
  <customwidget>
   <class>FileListWgt</class>
   <extends>QListWidget</extends>
   <header>../../../../../../code/ProjEd/headers/ProjectEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
